### ISSUE TITLE
Edit Cooldowns page in event that an Event handler was made.

### ISSUE THREAD

**Issue by CaptnLucid** [discordjs/guide#1518]
### Is your feature request related to a problem? Please describe.

The Cooldowns page provides code solely based on the Command Handler, but doesn't take into account that the Event Handler was mentioned before in the list.

### Describe the solution you'd like

I feel as though it would be considerate to add in a section on the cooldowns page for those who followed the Event Handling guide.

**Comment by tindolt** [comment]
I second this, having issues getting this to work. not sure what to do now.

**Comment by Jiralite** [comment]
It might be better to base it after the previous sections (so the event handler) instead of the command handler.

**Comment by Raigekijin** [comment]
> It might be better to base it after the previous sections (so the event handler) instead of the command handler, purely for this reason.

I disagree, as this part of following through is now broken due to incorrect commits.

**Comment by Jiralite** [comment]
That is not a valid reason. If there is a separate bug, that should be reported in itself.
